code/modules/battlemonsters/datum_elements.dm

```diff
@@ -0,0 +1,346 @@
+//General Rules for special effects:
+//Fire: Teamwork.
+//Energy: Can attack on the first turn or do things during the opponent's turn.
+//Water: After attack.
+//Defense: Benifits when defending.
+
+datum/battle_monsters/element
+	name = "neutral" //The name of the element
+	elements = BATTLE_MONSTERS_ELEMENT_NEUTRAL //Their general element.
+	//Note, values are normalized at the end of generation.
+	rarity = 1 //Relative rarity multiplier. Use for whether or not this card is picked for generation.
+	description = "%THEYRE neutral." //Description to add.
+	special_effects = "" //The special effects this element has, if any.
+	attack_type = BATTLE_MONSTERS_ATTACKTYPE_NONE //The attack flags of the monster. This is basically a general indicator of the weapon it uses.
+	defense_type = BATTLE_MONSTERS_ATTACKTYPE_NONE //The defense flags of the monster. This is basically a general indicator of what the monster is made out of. Not to be confused with their element.
+	rarity_score = 0 //Rarity score to add to this card.
+	//1 or lower means common, 2 means uncommon, 3 means rare, 4 or higher means legendary.
+	//Uncommon or higher rarities should be used sparringly, and only to cards that are super noteable.
+
+datum/battle_monsters/element/none
+	name = ""
+	id = "no_element"
+	icon_state = "no_element"
+	elements = BATTLE_MONSTERS_ELEMENT_NEUTRAL
+	description = ""
+	rarity = 5
+	rarity_score = -1
+
+datum/battle_monsters/element/trap //Indicator for trap cards.
```

Comment:
Please for the love of all that is holy add a `/` before datum.

Reply:
is there a code related reason or is it cosmetic related

Reply:
Searching and cosmetic